Tests/XcbeautifyLibTests/RendererTests/GitHubActionsRendererTests.swift

```diff
@@ -0,0 +1,676 @@
+        XCTAssertEqual(formatted, "Build target The Spacer of project Pods with configuration Debug")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCheckDependenciesErrors() {
+    }
+
+    func testCheckDependencies() {
+    }
+
+    func testClangError() {
+        let formatted = logFormatted("clang: error: linker command failed with exit code 1 (use -v to see invocation)")
+        XCTAssertEqual(formatted, "::error ::clang: error: linker command failed with exit code 1 (use -v to see invocation)")
+        XCTAssertEqual(parser.outputType, .error)
+    }
+
+    func testCleanRemove() {
+        let formatted = logFormatted("Clean.Remove clean /Users/admin/Library/Developer/Xcode/DerivedData/MyLibrary-abcd/Build/Intermediates/MyLibrary.build/Debug-iphonesimulator/MyLibraryTests.build")
+        XCTAssertEqual(formatted, "Cleaning MyLibraryTests.build")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCleanTarget() {
+        let formatted = logFormatted("=== ANALYZE TARGET The Spacer OF PROJECT Pods WITH THE DEFAULT CONFIGURATION Debug ===")
+        XCTAssertEqual(formatted, "Analyze target The Spacer of project Pods with configuration Debug")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCodesignFramework() {
+        let formatted = logFormatted("CodeSign build/Release/MyFramework.framework/Versions/A")
+        XCTAssertEqual(formatted, "Signing build/Release/MyFramework.framework")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCodesign() {
+        let formatted = logFormatted("CodeSign build/Release/MyApp.app")
+        XCTAssertEqual(formatted, "Signing MyApp.app")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testMultipleCodesigns() {
+        let formattedApp = logFormatted("CodeSign build/Release/MyApp.app")
+        let formattedFramework = logFormatted("CodeSign build/Release/MyFramework.framework/Versions/A (in target 'X' from project 'Y')")
+        XCTAssertEqual(formattedApp, "Signing MyApp.app")
+        XCTAssertEqual(formattedFramework, "Signing build/Release/MyFramework.framework")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCompileCommand() {
+    }
+
+    func testCompileError() {
+        let inputError = "/path/file.swift:64:69: error: cannot find 'input' in scope"
+        let outputError = "::error file=/path/file.swift,line=64,col=69::cannot find 'input' in scope\n\n"
+        XCTAssertEqual(logFormatted(inputError), outputError)
+        XCTAssertEqual(parser.outputType, .error)
+
+        let inputFatal = "/path/file.swift:64:69: fatal error: cannot find 'input' in scope"
+        let outputFatal = "::error file=/path/file.swift,line=64,col=69::cannot find 'input' in scope\n\n"
+        XCTAssertEqual(logFormatted(inputFatal), outputFatal)
+        XCTAssertEqual(parser.outputType, .error)
+    }
+
+    func testCompile() {
+#if os(macOS)
```

Comment:
Is there a Linux branch of this that should be tested as well?

Reply:
Since this is copied from `TerminalRendererTests`, I'm not sure what the expected output is. I can look at this in a follow-up PR, since this is an issue for all `OutputRendering` types.